common/types/shared.go

```diff
@@ -5925,6 +6109,10 @@ type RestartWorkflowExecutionRequest struct {
 	Identity          string             `json:"identity,omitempty"`
 }
 
+func (v *RestartWorkflowExecutionRequest) SerializeForLogging() (string, error) {
+	return SerializeRequest(*v)
```

Comment:
you can pass the pointer as the parameter, which will save a deep copy

Reply:
done